vertx-jooq-shared-reactive/src/main/java/io/github/jklingsporn/vertx/jooq/shared/reactive/UUIDAccessor.java

```diff
@@ -0,0 +1,16 @@
+package io.github.jklingsporn.vertx.jooq.shared.reactive;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.sqlclient.Row;
+
+import java.util.UUID;
+
+public class UUIDAccessor {
+    public static UUID getUUID(Row row, String field) {
+        Buffer buffer = row.getBuffer(field);
```

Comment:
The mysql implementation of the client doesn't implement getUuid (see https://github.com/eclipse-vertx/vertx-sql-client/blob/494d7c8321c8b0a8b33ab39ae72d1b91978a4ac5/vertx-mysql-client/src/main/java/io/vertx/mysqlclient/impl/MySQLRowImpl.java#L175 ). If you use a binary(16) and map it to a java UUID this works